xml/System.Linq/Enumerable.xml

```diff
@@ -11275,7 +11275,7 @@ If `count` is not a positive number, this method returns an identical copy of th
 ## Remarks
  This method returns zero if `source` contains no elements.
 
- The result does not include values that are `null`.
+ Items in `source` that are null are excluded from the computation of the sum. This method never returns `null`.
```

Comment:
I would probably combine this with the sentence above to say something like:

> Items in `source` that are `null` are excluded from the computation of the sum. This method returns zero if `source` contains no elements or all elements are `null`.

Reply:
@eiriktsarpalis 
Thanks for the feedback!
Done update the wording based on feedback above. Please review 🙂